src/components/Header/Header.tsx

```diff
@@ -8,25 +10,21 @@ import { MiddleHeader } from './MiddleHeader'
 import styles from './Header.module.scss'
 
 export const Header = () => {
-    const [menuBurgerActive, setMenuBurgerActive] = useState(false)
-
-    const onClickBurger = () => {
-        setMenuBurgerActive(preState => !preState)
-    }
+  const { isOpen: menuBurgerActive, onToggle: setMenuBurgerActive } = useToggle()
```

Comment:
setMenuBurgerActive  - again? does `setMenuBurgerActive ` set only Active ?

Reply:
be more attentive please! You do the same mistakes.

Reply:
fixed